aad/azure-spring-boot-sample-active-directory-b2c-resource-server/README.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
Please check whether the png file display well.

Reply:
yes, very well